package.json

```diff
@@ -1,37 +1,30 @@
 {
   "private": true,
   "devDependencies": {
-    "@types/express": "^4.0.35",
-    "@types/node": "^13.1.7",
-    "@types/normalize-url": "^1.9.1",
-    "@types/ws": "0.0.39",
+    "@types/express": "^4.17.13",
+    "@types/node": "^16.11.7",
     "@types/vscode": "1.65.0",
-    "css-loader": "^0.28.11",
-    "lerna": "^3.16.4",
-    "monaco-editor-core": "^0.33.0",
-    "rimraf": "^3.0.2",
-    "source-map-loader": "^0.2.3",
-    "style-loader": "^0.20.3",
-    "typescript": "^3.7.5",
-    "uglifyjs-webpack-plugin": "^1.2.4",
-    "webpack": "^3.11.0",
-    "webpack-merge": "^4.1.2",
-    "shx": "^0.3.4"
+    "editorconfig": "^0.15.3",
+    "monaco-editor": "^0.33.0",
```

Comment:
FYI, it seems that `^` has a different meaning on `0.x.x` versions: it will only allow patch update (`0.33.x`)

btw that's probably what we want

Reply:
Yes, true. This needs to be adjusted.